### ISSUE TITLE
Default i2c speed is 400kHz, conflicting with some breakouts

### ISSUE THREAD

**Issue by joezilla86** [adafruit/Adafruit_CircuitPython_FunHouse#19]
The default i2c generated by the Peripherals class is using a 400kHz speed. This causes errors when trying to use some sensor breakouts like PM25 that require a 100kHz i2c bus.

The Peripherals class makes a call to board.I2C(), but raises an error if an i2c bus is defined manually before initializing the FunHouse. 

This issue was [previously identified and resolved](https://github.com/adafruit/Adafruit_CircuitPython_CLUE/issues/36) for the CLUE board, but the change does not seem to affect the FunHouse. https://github.com/adafruit/Adafruit_CircuitPython_CLUE/issues/36

**Comment by caternuson** [comment]
What version of CircuitPython firmware are you using? The fix was done there, so should work for other libraries as well.

Quick test shows default clock is 100kHz.

```python
Adafruit CircuitPython 6.3.0 on 2021-06-01; Adafruit FunHouse with ESP32S2
>>> import board
>>> i2c = board.I2C()
>>> i2c.try_lock()
True
>>> i2c.scan()
[56, 119]
>>> 
```

![Screenshot from 2021-07-15 10-17-06](https://user-images.githubusercontent.com/8755041/125830070-cd6708a6-12c8-4663-90d6-cf99327c2085.png)

**Comment by joezilla86** [comment]
Maybe I was overzealous in saying the clock speed is wrong, I didn't actually verify. I'm away from the board at the moment, but I think it is on 6.3.??. But, the problem I was encountering is that if I tried to run an air quality sensor with the PM25 library, it would work for a few minutes and then return a checksum error. Looking into that I found a post that seemed to indicate it was an I2C clock speed issue. But when I try to define an I2C object with a 100kHz frequency, I get an error something like "board.SCL is already in use". And that happens if I create a manual i2c object before initializing the FunHouse too. I don't get errors if I just pass board.I2C() to the sensor initialization, but then the sensor doesn't work reliably.
